test/config/integration/certs/servercert_info.h

```diff
@@ -1,5 +1,6 @@
 // NOLINT(namespace-envoy)
-constexpr char TEST_SERVER_CERT_HASH[] = "DC:E2:2B:65:90:43:9A:36:1C:8E:6D:CA:42:8A:8C:37:C7:A1:77:"
-                                         "00:5B:C1:3E:33:8A:B9:2D:04:2C:B1:3F:0A";
-constexpr char TEST_SERVER_CERT_NOT_BEFORE[] = "Apr 7 16:46:35 2022 GMT";
-constexpr char TEST_SERVER_CERT_NOT_AFTER[] = "Apr 6 16:46:35 2024 GMT";
+constexpr char TEST_SERVER_CERT_256_HASH[] =
```

Comment:
regenerating this header file causes [CI failure](https://dev.azure.com/cncf/envoy/_build/results?buildId=125097&view=logs&j=8c169225-0ae8-53bd-947f-07cb81846cb5&t=6b0ace90-28b2-51d9-2951-b1fd35e67dec) since it overrides the old content, TEST_SERVER_CERT_NOT_AFTER is not generated but other test still replies on it.

I checked that servercert_info.h was first introduced by https://github.com/envoyproxy/envoy/pull/21428.
@daixiang0 How did you generate this file, I use generate_info_header in certs.sh, but the output is different.

Reply:
From a comment (https://github.com/envoyproxy/envoy/pull/21428/files#r884045881) it sounds like it was supposed to be generated, but it seems like that isn't the case for some reason. If you'd like you can manually add NOT_BEFORE and NOT_AFTER for the new correct values, and file an issue to fix so that they're generated automatically like they should be. Or if you'd like to fix it, that would be great.

Reply:
I see, it is probably that the command in certs.sh is modified downstream to generate header info but not get committed and upstreamed.

Reply:
> regenerating this header file causes [CI failure](https://dev.azure.com/cncf/envoy/_build/results?buildId=125097&view=logs&j=8c169225-0ae8-53bd-947f-07cb81846cb5&t=6b0ace90-28b2-51d9-2951-b1fd35e67dec) since it overrides the old content, TEST_SERVER_CERT_NOT_AFTER is not generated but other test still replies on it.
> 
> I checked that servercert_info.h was first introduced by #21428. @daixiang0 How did you generate this file, I use generate_info_header in certs.sh, but the output is different.

When run the script, it will update all contents, so you can add value and update all.

Reply:
@daixiang0 yes, but we should add changes to certs.sh otherwise when other people run certs.sh the old content is not generated.